tests/product/base_product_case.py

```diff
@@ -417,7 +424,7 @@ def replace_keywords(self, text, cluster=None, **kwargs):
             cluster = self.cluster
         test_keywords = {
             'rpm': self.presto_rpm_filename,
-            'rpm_basename': self.presto_rpm_filename[:-4],
+            'rpm_basename': self.presto_rpm_filename[:-4] + '.*',
             'rpm_basename_without_arch': self.presto_rpm_filename[:-11],
```

Comment:
Is this always going to be -11?  Is it because we always use the rpm for the same arch?